src/librustc_mir/borrow_check/nll/region_infer/error_reporting/region_name.rs

```diff
@@ -16,11 +16,22 @@ use rustc::infer::InferCtxt;
 use rustc::mir::Mir;
 use rustc::ty::subst::{Substs, UnpackedKind};
 use rustc::ty::{self, RegionVid, Ty, TyCtxt};
+use rustc::util::ppaux::HIGHLIGHT_REGION;
 use rustc_errors::DiagnosticBuilder;
 use syntax::ast::Name;
 use syntax::symbol::keywords;
 use syntax_pos::symbol::InternedString;
 
+fn with_highlight_region<R>(r: RegionVid, counter: usize, op: impl FnOnce() -> R) -> R {
```

Comment:
I'd probably put this code into ppaux and encapsulate the `HIGHLIGHT_REGION` variable itself

Reply:
Updated most recent commit to address this.